data-raw/qc-markdowns/temperature_data_prep.Rmd

```diff
@@ -607,8 +610,8 @@ Use CDEC data due to the longer time period and resolution of data. USGS can be
 * Note that Mill Creek has multiple temperature loggers to represent summer holding habitat. We will be acquiring this data and using in addition to the CDEC data. 
 
 ### CDEC
```

Comment:
Remaining TODO for Mill Creek is to apply threshold criteria using USGS data to the MLM CDEC data rather than the 37/-18 that are currently in the code.